src/endpoints/referral/click_count.rs

```diff
@@ -38,14 +38,14 @@ pub async fn handler(
     let mut output = Data { counts: vec![] };
     let mut i = 0;
     loop {
-        let start_naive_dt =
-            NaiveDateTime::from_timestamp_opt(query.since_day + i * query.spacing, 0).unwrap();
-        let start_time = Utc.from_utc_datetime(&start_naive_dt);
+        let _start_date_time =
+            DateTime::from_timestamp(query.since_day + i * query.spacing, 0).unwrap();
+        let start_time = Utc::now();
```

Comment:
Please reuse the original logic or something similar (Utc.from_utc_datetime(&start_naive_dt)), because this is not the same

Reply:
okay